index.bs

```diff
@@ -606,6 +701,39 @@ SessionResult = (StatusResult)
 
 ```
 
+<div algorithm>
+
+To <dfn lt="updating the event map">update the event map</dfn>, given a |list of
+event names|, a |list of contexts|, and a |target status|:
+
+  1. Let |events| be an empty set.
+
+    1. For each entry |event name| in the |list of event names|,
+       let |events| be the union of |events| and the result of [=trying=] to
+       [=obtain a set of events from a name=] with |event name|.
+
+    1. If the |list of contexts| is null, let |targets| be a set containing the [=global
```

Comment:
This case is if the `contexts` parameter is missing. Why shouldn't it be required, though?

Reply:
I think it makes sense to allow events to be enabled globally, and not passing in a list of contexts makes sense as a way to represent that.